packages/vue-flicking/demo/ssr/pages/features/Progress.vue

```diff
@@ -0,0 +1,198 @@
+    </flicking>
+    <div class="progress">
+      <div class="thumb" ref="thumb"></div>
+    </div>
+    <pre><code class="hljs html" data-script="flicking0">{{ code0 }}</code></pre>
+    <h1>panel.getProgress()</h1>
+    <ul class="extra">
+    <li>Indicates the progress of the relative index of each panel.</li>
+    </ul>
+    <flicking class="flicking flicking1"
+      :option="{
+        gap: 10,
+        circular: true,
+        moveType: { type: 'snap', count: 5 },
+      }"
+      @move="e => {
+        e.currentTarget.getAllPanels(true).forEach(panel => {
+          panel.getElement().innerHTML = panel.getProgress().toFixed(2);
+        });
+      }"
+      ref="flicking1">
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+    </flicking>
+    <div class="pagination pagination1"></div>
+    <pre><code class="hljs html" data-script="flicking1">{{ code1 }}</code></pre>
+
+    <h1>panel.getOutsetProgress()</h1>
+    <ul class="extra">
+    <li>When panel is completely invisible, outsetProgress becomes -1 at left(up), 1 at right(down) direction, and 0 when panel's anchor and hanger is overlapped.</li>
+    </ul>
+    <flicking class="flicking flicking2"
+      :options="{
+        gap: 10,
+        circular: true,
+        moveType: { type: 'snap', count: 5 }
+      }"
+      @move="e => {
+        const flicking = e.currentTarget;
+
+        flicking.getAllPanels(true).forEach(panel => {
+          panel.getElement().innerHTML = panel.getOutsetProgress().toFixed(2);
+        });
+      }">
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+    </flicking>
+    <div class="pagination pagination2"></div>
+    <pre><code class="hljs html" data-script="flicking2">{{ code2 }}</code></pre>
+
+    <h1>panel.getVisibleRatio()</h1>
+    <ul class="extra">
+    <li>Indicates the percentage of areas within the viewport that the panel occupies.</li>
+    </ul>
+    <flicking class="flicking flicking3"
+      :options="{
+        gap: 10,
+        circular: true,
+        moveType: { type: 'snap', count: 5}
+      }"
+      @move="e => {
+        const flicking = e.currentTarget;
+
+        flicking.getAllPanels(true).forEach(panel => {
+          panel.getElement().innerHTML = panel.getVisibleRatio().toFixed(2);
+        });
+      }">
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+      <div class="panel"></div>
+    </flicking>
+    <div class="pagination pagination3"></div>
+    <pre><code class="hljs html" data-script="flicking3">{{ code3 }}</code></pre>
+  </div >
+</template>
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+@Component({})
+export default class Progress extends Vue {
```

Comment:
@WoodNeck should initialize the progress status of DOM after mounted
  - Ref https://github.com/naver/egjs-flicking/blob/16bb2ae44f8cdd9f978382355da9f364bb0dd1f0/packages/react-flicking/src/demo/features/Progress.tsx#L111

Reply:
Also fixed this on e1d5a12, thanks.